Assets/MixedRealityToolkit/_Core/Definitions/Utilities/Handedness.cs

```diff
@@ -16,27 +16,27 @@ public enum Handedness : byte
         /// <summary>
         /// No hand specified by the SDK for the controller
         /// </summary>
-        None = 0 << 0,
+        None = 0,
         /// <summary>
         /// The controller is identified as being provided in a Left hand
         /// </summary>
-        Left = 1 << 0,
+        Left = 1,
         /// <summary>
         /// The controller is identified as being provided in a Right hand
         /// </summary>
-        Right = 1 << 1,
+        Right = 2,
         /// <summary>
         /// The controller is identified as being either left and/or right handed.
         /// </summary>
         Both = Left | Right,
         /// <summary>
-        /// Global catchall, used to map actions to any controller (provided the controller supports it)
-        /// *Note, by default the specific hand actions will override settings mapped as both
+        /// Reserved, for systems that provide alternate hand state.
         /// </summary>
-        Any = None | Left | Right,
+        Other = 4,
```

Comment:
dont care for this. what is an Other hand state? doesnt feel needed.... except to differentiate any from both.

both may actually be something very specific in the future (might not be decomposable into left and right parts).

Reply:
Well right now `Any` and `Both` are `3`, which is a problem

Reply:
true. maybe define Both as a discrete value and Any as Left | Right | Both

Reply:
Then what's the point of using the bit flags if we assign both as a discrete value?

Reply:
Any should also include None, as it relates simply to ANY input.
As for both, it relates to "Both hands" effectively.

Other is a special case for new / custom controllers that we simply don't have a definition for, but still allow to function and participate in the framework until it evolves to include it.

Other could be included in Any, but I don't think so due to it's custom nature, it needs to be treated in a special case.